install.sh

```diff
@@ -399,14 +392,14 @@ fi
 if ($INSTALL_NAV) && !(ls $CATKIN_DIR/src | grep --quiet "Navigator"); then
 	instlog "Downloading the Sub8 repository"
 	cd $CATKIN_DIR/src
-	git clone -q https://github.com/uf-mil/Sub8.git
+	git clone --recursive -q https://github.com/uf-mil/Sub8.git
 	cd $CATKIN_DIR/src/Sub8
 	instlog "Rolling back the Sub8 repository; do not pull the latest version!"
```

Comment:
This is for a Navigator installation. I have the script pull the version of Sub8 that was last known to work with that stack since we will no longer be supporting Indigo on the current Sub8.